profile/src/wallet_kit_common/has_placeholder.rs

```diff
@@ -0,0 +1,4 @@
+pub trait HasPlaceholder {
+    fn placeholder() -> Self;
```

Comment:
could possible be called: `HasPreviewValues` and `fn preview` and `fn preview_other`.